spec/syntax.md

```diff
@@ -526,18 +528,73 @@ unrecognized reserved sequences have no meaning and MAY result in errors during
 
 ```abnf
 reserved       = reserved-start reserved-body
-reserved-start = "!" / "@" / "#" / "%" / "*" / "<" / ">" / "/" / "?" / "~"
+reserved-start = "!" / "#" / "%" / "*" / "<" / ">" / "/" / "?" / "~"
 
 reserved-body  = *( [s] 1*(reserved-char / reserved-escape / quoted))
 reserved-char  = %x00-08        ; omit HTAB and LF
                / %x0B-0C        ; omit CR
                / %x0E-19        ; omit SP
-               / %x21-5B        ; omit \
+               / %x21-3F        ; omit @
+               / %x41-5B        ; omit \
                / %x5D-7A        ; omit { | }
                / %x7E-D7FF      ; omit surrogates
                / %xE000-10FFFF
 ```
 
+### Expression Attributes
+
+An **_<dfn>expression attribute</dfn>_** is a key-value pair
+containing a named argument that is attached to the _expression_ as a whole.
+They are intended to convey information about the _expression_
+that is not necessarily applicable as a _function_ _option_.
```

Comment:
I think this is too opinionated for us to include at this juncture. I would tone it down:

```suggestion
These might be used to convey information about the _expression_ that are not necessarily applicable as _options_ to a _function_, such as attributes related handling during the translation process.
Or they might be used to override contextual or implementation-specific values used by a _function_ during formatting, such as the locale, time zone, or other settings.
```